arviz/plots/backends/bokeh/autocorrplot.py

```diff
@@ -59,10 +60,8 @@ def plot_autocorr(
         ax.title = title
 
     if axes.size > 0:
-        axes[0, 0].x_range._property_values["start"] = 0  # pylint: disable=protected-access
-        axes[0, 0].x_range._property_values["end"] = max_lag  # pylint: disable=protected-access
-        axes[0, 0].y_range._property_values["start"] = -1  # pylint: disable=protected-access
-        axes[0, 0].y_range._property_values["end"] = 1  # pylint: disable=protected-access
+        axes[0, 0].x_range = DataRange1d(bounds="auto")  # pylint: disable=protected-access
```

Comment:
Hi, instead of using `bounds="auto"` maybe use `rcParam.bokeh.bounds` and set it to `auto`. This way user can change this manually by changing that rcParam. E.g. `(0, None)`.

Reply:
Yeah that makes sense. Should I also add a ```_validate_tuple``` function to check for valid tuple value?

Reply:
That sounds reasonable